Server/VikingVault.API/Controllers/PaymentTogglingController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using VikingVault.Services;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.API.Controllers
+{
+   
+    [Route("[controller]")]
+    [ApiController]
+    public class PaymentTogglingController : ControllerBase
+    {
+        private readonly IPaymentTogglingService _paymentTogglingService;
+
+        public PaymentTogglingController(IPaymentTogglingService paymentTogglingService)
+        {
+            _paymentTogglingService = paymentTogglingService;
+        }
+
+        [HttpGet("{id}", Name = "Get")]
```

Comment:
How does the endpoint URL look like when you call this method?

Reply:
localhost:bla/paymenttoggling/{id}

Reply:
And what does that "Name=Get" do?